commitizen/cli.py

```diff
@@ -133,6 +133,23 @@
                     },
                 ],
             },
+            {
+                "name": "undo",
+                "help": "revert bump or commit",
+                "func": commands.Undo,
+                "arguments": [
+                    {
+                        "name": ["--bump"],
```

Comment:
What do you think of adding a shortcut here? `-b` and `-c` for commit?
We'd have `cz undo -b`

Reply:
Sounds great 💯